test/bst.t

```diff
@@ -415,17 +365,12 @@ Test pids.max cgroup usage
 	$ ! bst --cgroup /sys/fs/cgroup/bst --no-cgroup-remount --limit pids.max=-1 cat /sys/fs/cgroup/bst/pids.max 2>/dev/null
 
 Test mutliple climits in conjunction
-
-	$ bst --cgroup=/sys/fs/cgroup/bst --no-cgroup-remount --limit cpu.max="5000 7000" --limit cpu.weight=100 --limit memory.max=1G --limit memory.low=1M --limit memory.max=1G --limit memory.swap.max=1G --limit io.max="8:0 rbps=100 wbps=100 riops=100 wiops=100" --limit io.latency="8:0 target=20" --limit pids.max=50 cat /sys/fs/cgroup/bst/cpu.max; cat /sys/fs/cgroup/bst/cpu.weight; cat /sys/fs/cgroup/bst/memory.min; cat /sys/fs/cgroup/bst/memory.low; cat /sys/fs/cgroup/bst/memory.high; cat /sys/fs/cgroup/bst/memory.max; cat /sys/fs/cgroup/bst/memory.swap.high; cat /sys/fs/cgroup/bst/memory.swap.max; cat /sys/fs/cgroup/bst/io.weight; cat /sys/fs/cgroup/bst/io.max; cat /sys/fs/cgroup/bst/io.latency; cat /sys/fs/cgroup/bst/pids.max
+	$ bst --cgroup=/sys/fs/cgroup/bst --no-cgroup-remount --limit cpu.max="5000 7000" --limit cpu.weight=100 --limit memory.low=1M --limit memory.max=1G --limit pids.max=50 --workdir /sys/fs/cgroup/bst cat cpu.max cpu.weight memory.min memory.low memory.high memory.max io.weight pids.max
```

Comment:
I think I'd also rather have this be on multiple lines for readability.

Reply:
Reformatted tests.